docs/source/usage/parameter_scans.rst

```diff
@@ -12,11 +12,13 @@ Each workflow consists of a ``Snakefile`` in which the workflow is defined using
 How to use
 """"""""""
 
-The example presented here performs several simulations with given input parameters.
+The example presented here performs several simulations with given input parameters with the LaserWakefield example of PIConGPU on hemera.
+
+#. Copy the ``Snakefile`` and ``config.yaml`` which are located in the PIConGPU source code here: ``picongpu/share/picongpu/examples/LaserWakefield/lib/python/snakemake/``. There also a ``requirements.txt`` and some exemplary parameters ``params.csv`` can be found which are compatible with the LaserWakefield example.
```

Comment:
Missing comma and drop the 'also' please. You might want rephrase along the lines of 'In addition, a .... can be found there, which are ...'